tools/validate_spec.py

```diff
@@ -131,18 +133,21 @@ def get_message_to_validate(message_path: Path, is_valid: bool) -> "OriginalMess
             for message_file_path in path_generator:
                 msg = get_message_to_validate(message_file_path, path == path_valid)
                 validation_result = validator.validate_message(msg)
-                json_output_writer.write(validation_result.get_json_output())
-                print(validation_result.get_abbreviated_output())
+                json_output_writer.write(validation_result)
                 if validation_result.classification in [
                     Classification.FP,
                     Classification.FN,
                 ]:
+                    print(f"{message_file_path}: FAILED")
+                    print(f"{validation_result.get_abbreviated_output()}\n")
```

Comment:
tests/data/ethernet/invalid/ethernet_undefined.raw: FAILED
classification: FalseNegative, identifier: Message could not be parsed, file name: ethernet_undefined.raw, error: PyRFLXError: pyrflx: error: none of the field conditions ['Type_Length <= 1500', 'Type_Length >= 1536'] for field Type_Length have been met by the assigned value: 0000010111101011

tests/data/ethernet/valid/ethernet_802.3.raw: FAILED
classification: FalsePositive, identifier: Ethernet::Frame, file name: ethernet_802.3.raw, error:

---

1. The file name is redundant.
2. `error:` is empty in some cases.
3. `error: PyRFLXError: pyrflx: error:` looks quite strange.
4. IMHO these obvious prefixes like `classification: ` are unnecessary.
5. Are there cases where the identifier is different from the given parameter `-m`?
6. IMHO putting everything on one line makes it badly readable. Line breaks would help here.

Reply:
> error: is empty in some cases.
I think you ran `test_validation_negative`. In this case, the supplied message is valid, but was provided as invalid. Thus, the validation fails, but no error message is printed, as no error is thrown while parsing.

Reply:
>Are there cases where the identifier is different from the given parameter -m?

If a provided message is invalid (could not be parsed), `identifier` would indicate that in the output with `identifier: Message could not be parsed`.

Reply:
>IMHO these obvious prefixes like classification: are unnecessary.

I like the prefixes as they provide structure to the output. Just printing `FalseNegative` on its own line would feel a bit lost in my opinion. 
I agree that `error: PyRFLXError: pyrflx: error:` are too many prefixes.

Reply:
> I think you ran `test_validation_negative`. In this case, the supplied message is valid, but was provided as invalid. Thus, the validation fails, but no error message is printed, as no error is thrown while parsing. 

You are right. That doesn't answer why `error: ` is shown if there is no message.

Reply:
> If a provided message is invalid (could not be parsed), `identifier` would indicate that in the output with `identifier: Message could not be parsed`.

How is `Message could not be parsed` related to `identifier`?

Reply:
> I like the prefixes as they provide structure to the output. Just printing `FalseNegative` on its own line would feel a bit lost in my opinion. 

IMHO the prefixes are just useless noise. The output should be short and precise. Prefixes would be only helpful if the output could be misinterpreted without them.

Reply:
Created https://github.com/Componolit/RecordFlux-specifications/issues/28 to address the prefixes in the console output separately.

Reply:
I suppose the remaining points (1-2, 4-6) will be fixed in this PR?

Reply:
Yes, I have a fix ready and will push it shortly.

Reply:
Changed console output in 35f5533